### PR TITLE
fix: add chart.js to public assets for eye exam

### LINKED ISSUE #7155: bug: eye exam chart fails since upgrade to latest chart.js package
### Describe the bug

https://community.open-emr.org/t/chart-does-not-appear-in-glaucoma-zone/18361

### To Reproduce

1. Add a '....'

### Client configuration

- Browser:
- OpenEMR version:
- Operating system:

### PR DESCRIPTION
**Pull request by stephenwaite**
Fixes #7155

#### Changes proposed in this pull request:
using chart.umd.js to get around type='module' in original script include
also increases size of OD/OS fields

### CODE REVIEW

**Comment by stephenwaite** [comment]
Screenshot from 2024-01-08 16-11-55

:star_struck:

**Comment by stephenwaite** [comment]
uncovered some other console errors
Screenshot from 2024-01-08 16-12-48

File: interface/forms/eye_mag/php/eye_mag_functions.php [hunk 1]
```diff
@@ -5309,7 +5309,6 @@ function display_GlaucomaFlowSheet($pid, $bywhat = 'byday')
 
             </table>
         </div>
-        <script type="module" src="<?php echo $GLOBALS['assets_static_relative'] ?>/chart.js/dist/chart.js"></script>
```
Comment (bradymiller) [inline interface/forms/eye_mag/php/eye_mag_functions.php:5312]: fare thee well :)

**Review by bradymiller — APPROVED** [review]

**Review by bradymiller — APPROVED** [review]

**Comment by stephenwaite** [comment]
have to fix the conflict and going to style the chart config and then bring in

**Comment by bradymiller** [comment]
2DV

**Pull request merged**